nixos/modules/services/security/sshguard.nix

```diff
@@ -77,65 +78,65 @@ in {
             Systemd services sshguard should receive logs of.
           '';
       };
-
     };
-
   };
 
-
   ###### implementation
 
   config = mkIf cfg.enable {
 
-    environment.systemPackages = [ pkgs.sshguard pkgs.iptables pkgs.ipset ];
```

Comment:
Is `sshguard` useless on the command line? Is that the reason for the removal of this?

Reply:
Correct, you don't interact with sshguard. You could argue that ```ipset``` should be there in case you want to manually inspect the ipset that sshguard manipulates but for all normal uses, neither of these are needed.